README.md

```diff
@@ -177,6 +177,10 @@ vous trouverez sur cette plateform des cours et examens qui vous accompagnerons
 * [forcoder.org](http://forcoder.org/) IT ebooks and e-Learning videos for download
 * [Packtpub](https://www.packtpub.com/) Free ebook available daily
 
+### Learn Languages
+* [Duolingo](https://www.duolingo.com/): The world's best way to learn a language!
+* [Learn a Language](https://www.learnalanguage.com/): Learn a language with hundreds of free lessons
+
```

Comment:
I suggest adding this at the bottom of the list, also update the summary

Reply:
Done on my fork, but could you show me how I could modify my recent commit instead of creating a new one ?

Reply:
just push to your branch

Reply:
Done